lib/tests/test_lexer.c

```diff
@@ -128,53 +110,77 @@ _current_lloc(void)
 }
 
 #define assert_token_type(expected)                                     \
-  assert_gint(_current_token()->type, expected, "Bad token type at %s:%d", __FUNCTION__, __LINE__);
+  cr_assert_eq(_current_token()->type, expected, "Bad token type at %s:%d", __FUNCTION__, __LINE__);
 
 #define assert_parser_string(expected)                          \
   _next_token();                                                        \
   assert_token_type(LL_STRING);                                        \
-  assert_string(_current_token()->cptr, expected, "Bad parsed string at %s:%d", __FUNCTION__, __LINE__); \
+  cr_assert_str_eq(_current_token()->cptr, expected, "Bad parsed string at %s:%d", __FUNCTION__, __LINE__); \
 
-#define assert_parser_block(expected) \
-  _next_token();                                                        \
-  assert_token_type(LL_BLOCK);                                         \
-  assert_string(_current_token()->cptr, expected, "Bad parsed string at %s:%d", __FUNCTION__, __LINE__);
+static void
+assert_parser_block(gchar *expected)
+{
+  _next_token();
+  assert_token_type(LL_BLOCK);
+  cr_assert_str_eq(_current_token()->cptr, expected, "Bad parsed string at %s:%d", __FUNCTION__, __LINE__);
```

Comment:
This is another occurrence for broken expansion.  Please reevaluate the patch and cleanup all the other places.

Reply:
ok. Rechecking all and committing again.